### ISSUE TITLE
Where do agent frameworks fit?

### ISSUE THREAD

**Issue by skrawcz** [lfai/lfai-landscape#795]
Hi.

I think landscape needs a new category around frameworks that help one build agents.

For example https://github.com/dagworks-inc/burr. Burr could technically go under workflow frameworks, but it's not a replacement for the other frameworks listed there.

Thoughts?

**Comment by ibrahimhaddad** [comment]
We can def open a new category. The requirement is to have at least 300 stars. Are there other similar projects to Burr that we can list if we open this category? thx.

**Comment by skrawcz** [comment]
> We can def open a new category. The requirement is to have at least 300 stars. Are there other similar projects to Burr that we can list if we open this category? thx.

Yep there's quite a few, and they run the spectrum of "all in one" to build your own. Burr is on the latter side.

**Comment by jmertic** [comment]
@skrawcz You can propose new categories as part of a PR in the `landscape.yml` file. The data format for the entry can be found at https://github.com/cncf/landscape2/blob/main/docs/config/data.yml.

Closing for now; feel free to re-open if this is still a concern. Thanks!
